base/string_utils.cpp

```diff
@@ -326,4 +326,29 @@ bool AlmostEqual(string const & str1, string const & str2, size_t mismatchedCoun
   return false;
 }
 
+void Split(string const & s, char delimiter, vector<string> & target)
```

Comment:
No, `Tokenize()` function does a similar thing, but different. For example, it groups delimiters. It cannot take a `char`, but requires a C string (which would require additional conversion from `char`). It is much harder to understand than `std::sort`. The tokenizer is awfully hard to use if you need something other than `SimpleTokenizer`. Finally, this function is obviously faster.